docs/scheduledtasks.md

```diff
@@ -0,0 +1,25 @@
+### Scheduled Tasks
+The Framework is responsible for executing all Open vStorage scheduled tasks. These tasks are kicked of by celery beat, the task scheduler of Celery. To schedule the exact timing of these tasks, celery schedules are used. These schedules are store in the [distributed configuration management system](configmgmt.md).  
```

Comment:
The default schedule is not in configuration management, but hardcoded in code. The configuration management merely allows a user to change the schedule or disable it.